start/server/resolvers.js

```diff
@@ -17,8 +19,14 @@ const getAllBooks = () => {
   return books;
 };
 
-const addBook = (title, name) => {
-  books.push({ title: title, author: {name: name }});
+const addBook = (title, name, id) => {
+  books.push({ id: id, title: title, author: {name: name }});
```

Comment:
Instead of passing the id, can we add code to figure that out?  Think of the front end being concerned with the title and author, but the backend is concerned with the id as it should be unique.  That way if multiple people users are adding an id, we don't duplicate them.